### PR TITLE
[Task#68292]Add accept registration functionality

### PR DESCRIPTION
**Pull request by tungbd-0214**
## Related Tickets:
- [#68292](https://edu-redmine.sun-asterisk.vn/issues/68292)

## WHAT this PR does?
- Add accept registration functionality

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.
Screen Shot 2023-09-15 at 16 48 03

Screen Shot 2023-09-15 at 16 48 46

### CODE REVIEW

**Comment by tungbd-0214** [comment]
ready

File: app/Http/Controllers/RegistrationController.php [hunk 1]
```diff
@@ -8,13 +8,23 @@
...
+            [
             'registrations' => Registration::all(),
-        ]);
+            ]
+        );
+    }
+
+    public function update(Request $request, Registration $registration)
+    {
+        $registration->update(['status'=> 1]);
```
Comment (ghost) [inline app/Http/Controllers/RegistrationController.php]: function này k return gì à em?

File: app/Http/Controllers/RegistrationController.php [hunk 2]
```diff
@@ -8,13 +8,23 @@
...
+    }
+
+    public function destroy(Registration $registration)
+    {
+        $registration->delete();
```
Comment (ghost) [inline app/Http/Controllers/RegistrationController.php]: này cũng v

**Comment by ghost** [comment]
fail CI

**Comment by tungbd-0214** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**